src/main/java/com/setycz/chickens/jei/ChickensJeiPlugin.java

```diff
@@ -97,7 +97,7 @@ public void onRuntimeAvailable(IJeiRuntime jeiRuntime) {
     private List<BreedingRecipeWrapper> getBreedingRecipes() {
         List<BreedingRecipeWrapper> result = new ArrayList<BreedingRecipeWrapper>();
         for (ChickensRegistryItem chicken : ChickensRegistry.getItems()) {
-            if (chicken.getTier() > 1) {
+            if (chicken.getTier() > 1 && chicken.getParent1 != null && chicken.getParent2 != null) {
```

Comment:
[getTier](https://github.com/setycz/ChickensMod/blob/1.10.2/src/main/java/com/setycz/chickens/ChickensRegistryItem.java#L96) already checks for parents not being null. How could this fix any issue